.github/workflows/check-vulnerabilities.yml

```diff
@@ -36,4 +36,4 @@ jobs:
           go-version-input: "1.21"
           go-package: ./...
           check-latest: true
-          cache: true
+          cache: false
```

Comment:
We don't want the built in caching (which uses lock files) to mess up our caches that are stored based on non-lockfiles for the main testing actions. These are quick actions that I don't care much about, the main one we care about shouldn't miss caches because some other action cached a lock-file based one.

Reply:
Another note: The places I added `cache: false` to where it didn't exist is because since the upgrade to the new version the default was changed to `cache: true`.